frontend-api/src/main/java/uk/gov/di/authentication/frontendapi/lambda/LoginHandler.java

```diff
@@ -34,7 +37,8 @@
         implements RequestHandler<APIGatewayProxyRequestEvent, APIGatewayProxyResponseEvent> {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(LoginHandler.class);
-
+    private final CodeStorageService codeStorageService;
+    private static final int MAX_INCORRECT_PASSWORD_ATTEMPTS = 5;
```

Comment:
You should get this value from config such as `configurationService.getCodeMaxRetries()`